src/containers/HomePage/styles.js

```diff
@@ -1,8 +1,16 @@
 import { makeStyles } from '@material-ui/core'
-import { ACCENT2_COLOR, ACCENT_COLOR } from '../../core/values/colors'
+import {
+  ACCENT2_COLOR,
+  ACCENT_COLOR,
+  THIRD_COLOR,
+} from '../../core/values/colors'
 
 export const useStyles = makeStyles(() => ({
```

Comment:
@rostislavborovyk, can you place styles after components. Also, why export them?